docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -857,11 +857,17 @@ include::ingest-node-common-processor.asciidoc[]
 
 [source,js]
 --------------------------------------------------
+PUT _ingest/pipeline/studentindex
```

Comment:
Naming a pipeline "*index" might be misleading here, as it obscures the fact that is is just a pipeline id. Also I would probably not introduce a new name here, expecially since we don't test this snippet.

Reply:
Good point. How about `my-pipeline-id` which is already there on the top of the whole doc?

Reply:
+1